src/xo-app/settings/remotes/index.js

```diff
@@ -218,6 +218,27 @@ export default class Remotes extends Component {
 
   _handleRemoteTypeSelection = type => this.setState({type})
 
+  _checkNameExists = () => {
+    const newName = this.refs.name.value
+    const remotes = this.props.remotes
+    let exists = false
+
+    const title = <span>
+      <Icon icon={'error'} />
+      {' '}
+      {_('remoteTestName')}
+    </span>
+    const body = <p>
+      {_('remoteTestNameFailure')}
+    </p>
+
+    map(remotes, values => map(values, remote => {
```

Comment:
Use map only to create a new array from a collection.

What you want is `forEach()` and `find()`.